spec/controllers/users/confirmations_controller_spec.rb

```diff
@@ -9,7 +9,7 @@
     it 'tracks nil email confirmation token' do
       analytics_hash = {
         success: false,
-        error: "Confirmation token can't be blank",
+        error: 'Confirmation token Please fill in this field.',
```

Comment:
the text for this needs to be tweaked at least -- looks like a control find & replace oversight